File: Cpp_Programming/Competative/Assignments/Assignment_2/program5.cpp
//Accept number from user and check wheather number is even or odd

#include<iostream>
using namespace std;

typedef int BOOL;
#define TRUE 1
#define FALSE 0

BOOL CheckEven(int iNo)
{
    if((iNo % 2) == 0)
    {
        return TRUE;
    }
    else
    {
        return FALSE;
    }
}

int main()
{
    int iValue = 0;
    BOOL bRet = FALSE;

    cout<<"Enter the number\n";
    cin>>iValue;

    bRet = CheckEven(iValue);

    if(bRet == TRUE)
    {
        cout<<"Number is Even\n";
    }
    else
    {
        cout<<"Number is odd\n";
    }

    return 0;
}

File: Java_Programming/Competative/Assignments/Assignment_2/program5.java
package Competative.Assignments.Assignment_2;

//Accept number from user and check wheather number is even or odd

import java.util.*;

class DisplayX
{
    boolean CheckEven(int iNo)
    {
        if((iNo % 2) == 0)
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

class program5 
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;
        boolean bRet = false;

        System.out.println("Enter the number");
        iValue = sobj.nextInt();

        DisplayX dobj = new DisplayX();

        bRet = dobj.CheckEven(iValue);

        if(bRet == true)
        {
            System.out.println("Number is Even");
        }
        else
        {
            System.out.println("Number is odd");
        }
    }
}
